index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+</head>
+    <main class="box">
+    <header class="header outer-container">
+        <nav>
+            <ul>
+                <li><a href="">Case studies</a></li>
+                <li><a href="">Services</a></li>
+                <li><a href="">Contact Us</a></li>
+            </ul>
+        </nav>
+
+    <div class="inner-container">
+    <h1>Your website is an investment in the success of your business.</h1>
+    <h2>We are here to help you succeed.</h2>
+
+    <button>Let's work together</button>
+        <img class="squiggle-stripe" src="./assets/assets-for-concept/squiggle_line_stripe.svg" alt="squiggle-line-stripe">
+    </div>
+    </header>
+<body>
+<br>
+<br>
+<section>
+    <div class="intro">
+        <h2>Rekupe is a digital agency in Los Angeles.
+            <br>
+            <br>
+        We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.
+            <br>
+            <br>
+        We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</h2>
+        <br>
+        <br>
+        <br>
+    <p>Have a question? Click here to contact us!</p></div>
+    <br>
+    <br>
+</section>
+<section class="section-projects">
+    <article>
+    <div class="projectRolingStone">
+        <span class="image-wrapper">
+        <img src="./assets/assets-for-concept/work-image-01.jpeg" alt="Rolling-stone-laptop"></span>
+    </div>
+        <div>
+        <p>Rolling Stone</p>
+            <img class="squiggle-line" src="./assets/assets-for-concept/squiggle_line.svg" alt="squiggle-line">
+        <p>You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global icon.</p>
+    </div>
+    </article>
+
+    <article>
+    <div>
+    <p>Variety</p>
+        <div>
+    <img class="squiggle-line" src="./assets/assets-for-concept/squiggle_line.svg" alt="squiggle-line">
+            </div>
+        <span class="image-wrapper">
+        <img src="./assets/assets-for-concept/work-image-02.jpeg" alt="variety-book-laptop"></span>
+    <p>Variety is an industry leader for breaking entertainment news, reviews and box office results with archive of content dating back to 1905.</p>
+    </div>
+    </article>
+
+    <article>
+    <div>
+    <p>Worth</p>
+    <p>A global media platform connecting a community that embraces worth beyond wealth.</p>
+    </div>
+    </article>
```

Comment:
De articles missen nog een `display: flex` i.c.m. `align-items: center` waardoor de boel netjes in het midden naast elkaar staat